community/codegen/src/main/java/org/neo4j/codegen/CodeBlock.java

```diff
@@ -73,26 +87,31 @@ private void endBlock()
         }
     }
 
+    protected void emit( Consumer<MethodEmitter> emitFunction )
+    {
+        emitFunction.accept( emitter );
+    }
+
```

Comment:
This indirection looks like it was added in order to defer the actions. It was used in `TryBlock`. It seems like then the overriding of this method has been removed, rendering this indirection unnecessary and costly (extra object creation, extra indirection).